stable/localvolumeprovisioner/templates/daemonset.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
 {{- end }}
-      priorityClassName: {{ .Values.priorityClassName }}
+      # priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
Should we leave this in rather than comment it out, just with the understanding for now that it needs to be `kube-system` deployed?